core/server/src/main/java/alluxio/master/AbstractMaster.java

```diff
@@ -204,8 +205,13 @@ public void stop() throws IOException {
   }
 
   @Override
```

Comment:
The masters should be stopped when they are called, so no journal entries should be in progress. I've updated the javadoc to include this